src/Api/DataProvider/AbstractElasticSearchDataProvider.php

```diff
@@ -244,15 +246,48 @@ protected function logStatistics(string $type, array $identifiers, array $result
     {
         $className = substr(\get_class($this), strrpos(\get_class($this), '\\') + 1);
 
+        $imageUrls = is_array($results) ? $this->getImageUrls($results) : [];
+
         $this->statsLoggingService->info('Cover request/response', [
             'service' => $className,
             'clientID' => 'REST_API',
             'remoteIP' => $request->getClientIp(),
             'isType' => $type,
             'isIdentifiers' => $identifiers,
-            'fileNames' => $this->getImageUrls($results),
+            'fileNames' => !empty($imageUrls) ? array_values($imageUrls) : null,
```

Comment:
Why not default to [] and not NULL then the data structure is the same.

Reply:
$this->getImage() returned null or array with entries before. 
So to not change the logging structure of "fileNames", I chose this solution, after I changed the return type of getImageUrls() from mixed to array.

Reply:
We are changing the data structure... I think ES would be happier if the data always was an array.